README.md

```diff
@@ -1,2 +1,43 @@
 # brandingUpdate
 
+
+The following table provides a summary of the terms that are changing in accordance with the recent branding guidelines.
+
+When directly referenced in the documentation, all of the terms listed below should be updated to their rebranded counterparts as capabilities of Adobe Experience Platform.
+
+| Original Term  |Updated Term | Explanation  |Previous Example Sentence  | Current Branded Use |
+|---|---|---|---|---|
+|  **Platform Launch Server Side** |  event forwarding | Launch Server Side is now event forwarding. It is written in lower case as it is a capability not a service.  | See the **Launch Server Side** overview and Getting started with Experience **Platform Launch Server Side** for more information.  | See the **event forwarding overview** and Getting started with **event forwarding** for more information.  |
```

Comment:
The note re: having this term written lower-case applies to all of these term changes. Add a note to the intro paragraphs that explains this rather than nesting it in this one table cell. Clarify the point that these are no longer proper nouns and instead should be written as lowercase, generic capabilities. I would include the comparison we make to "schemas" to help illustrate this before we delve into the details of each term